### PR TITLE
WRN-18219: [Drawing POC] - Styling

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Applied proper styling for drawing component.

### Links
[//]: # (Related issues, references)
WRN-18219

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: Drawing/Drawing.module.less [hunk 1]
```diff
@@ -3,12 +3,47 @@
...
+	.drawingTools {
+		margin: 0;
+
+		& > * {
+			font-size: 36px;
+		}
+	}
+
+	.button {
+		min-width: 3rem;
```
Comment (daniel-stoian-lgp) [inline Drawing/Drawing.module.less:43]: please use px

File: Drawing/ColorPicker.module.less [hunk 2]
```diff
@@ -4,15 +4,21 @@
...
+		padding-left: 21px;
 	}
 
 	.coloredButton {
 		transform: none;
 		border-radius: 999px;
-		border: 3px solid pink;
+		border: 3px solid white;
+		height: 71px;
+		min-width: 71px;
```
Comment (daniel-stoian-lgp) [inline Drawing/ColorPicker.module.less:20]: use multiple of 3

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Pull request merged**